### ISSUE TITLE
import issue with parser submodule in bh2err

### ISSUE THREAD

**Issue by BigPeet** [BigPeet/vim-axivion-bauhaus#19]
How to reproduce:

1. In the top-level directory
2. start python3
3. import bh2err

Result (for Python 3.6.9):
```bash
Python 3.6.9 (default, Jul 17 2020, 12:50:27) 
[GCC 8.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import bh2err
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/pwolf/.dotfiles/vim/vim/bundle/vim-axivion-bauhaus/bh2err/__init__.py", line 1, in <module>
    from bh2err.convert import convert_file, convert_text
  File "/home/pwolf/.dotfiles/vim/vim/bundle/vim-axivion-bauhaus/bh2err/convert.py", line 3, in <module>
    from bh2err.parsers import Parsers
  File "/home/pwolf/.dotfiles/vim/vim/bundle/vim-axivion-bauhaus/bh2err/parsers/__init__.py", line 1, in <module>
    from bh2err.parsers import sv_parser, mv_parser, av_parser, de_parser, cl_parser, cy_parser
  File "/home/pwolf/.dotfiles/vim/vim/bundle/vim-axivion-bauhaus/bh2err/parsers/sv_parser.py", line 1, in <module>
    import bh2err.parsers.base_parser as base_parser
AttributeError: module 'bh2err' has no attribute 'parsers'
```

Also happens when calling "vim" from anywhere.

**Comment by BigPeet** [comment]
This seems to be an issue with Python 3.6

**Comment by BigPeet** [comment]
fixed with 34ee57ae411e535cc54b0f707a71974c7f574fa9
